lib/helpers.js

```diff
@@ -9,9 +9,28 @@ const PRETTIER_CORE_OPTIONS = [
   'trailingComma',
   'bracketSpacing',
 ];
+const CORE_OPTIONS_ATOM_MAP = {
+  printWidth: 'editor.preferredLineLength',
+  tabWidth: 'editor.tabLength',
+};
 
 // helpers
-const getConfigOption = key => atom.config.get(`prettier-atom.${key}`);
+const isFileTypeSupported = path => SUPPORTED_FILE_TYPES.some(ext => path.endsWith(ext));
```

Comment:
I think this is the only line that got messed up by my other PR. I switched to using scopes so we don't need this anymore